app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -298,11 +320,18 @@ const CallScreen = ({
     usePermissionsChecker(micPermissionsGranted);
     const [showControlsInLandscape, setShowControlsInLandscape] = useState(false);
     const [showReactions, setShowReactions] = useState(false);
+    const callsTheme = useMemo(() => makeCallsTheme(theme), [theme]);
+    const style = getStyleSheet(callsTheme);
```

Comment:
as `getStyleSheet` is not using `makeStyleSheetFromTheme` this is not being memoized creating a new style on every render, either use `makeStyleSheetFromTheme` or memoize this